schemas/chemotherapy.json

```diff
@@ -50,18 +50,24 @@
       }
     },
     {
-      "name": "chemotherapy_drug_name",
-      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "name": "drug_rxnormid",
```

Comment:
I believe this is missing required restriction, and if you want to add the primaryId meta field, same in all files below

Reply:
@blabadi i removed the restriction bc in the ticket we said users could add `drug_name` OR `rx_normid` , which would need a script validation added.  If `or` is the case, then both should not be required. 

Is this a primary id?  I don't think it is.